src/lib/progress-bar/progress-bar.ts

```diff
@@ -9,9 +9,12 @@ import {
   Component,
   ChangeDetectionStrategy,
   ElementRef,
+  Inject,
   Input,
+  Optional,
   ViewEncapsulation
 } from '@angular/core';
+import {ANIMATION_MODULE_TYPE} from '@angular/platform-browser/animations';
```

Comment:
Wouldn't adding this import be considered a breaking change, in the case where somebody might not have the animations package? I think we had a discussion along the same lines with @mmalerba where we considered adding animations to a component, that didn't have any, a breaking change. That being said, most people should have `platform-browser` already.

Reply:
I think this is different because it doesn't require a module to be loaded; for animations, the breaking change is that you need to include `BrowserAnimationsModule` at the root for it to work. For this, I don't think it actually breaks anything.